README.md

```diff
@@ -41,98 +38,98 @@ automatically re-entered;
 - inband reauthentication is not supported; expiring tokens will trigger a disconnection and resume of a realtime
 connection.
 
-As with the REST API, it is intended that this library is upgraded incrementally and brought into line with the 1.1
+As with the REST API, it is intended that this library is upgraded incrementally and brought into line with the 1.2
 specification. If there are features that are currently missing that are a high priority for your use-case then please
 [contact Ably customer support](https://support.ably.io). Pull Requests are also welcomed.
 
 ## Using the Realtime API
 
 ### Creating a client
 
+<!-- GO IMPORT "context" -->
+
+<!-- GO EXAMPLE
+ctx := context.Background()
+-->
+
 ```go
-client, err := ably.NewRealtime(ably.NewClientOptions("xxx:xxx"))
+client, err := ably.NewRealtime(ably.WithKey("xxx:xxx"))
```

Comment:
I'm not sure we should overwhelm the README with error paths. For a full reference, users can check the Ably docs and the Go package docs.

Reply:
> I'm not sure we should overwhelm the README with error paths

This is a pretty important error path for users to be aware of though, no?

> For a full reference, users can check the Ably docs and the Go package docs.

Ok, can we at least put a link here to where those exist for handling connection errors?

Reply:
See cbdaa10.

Those examples are how things _should_ be, but they won't work at least until #172 is merged, and they probably depend on [RTL6 too](https://github.com/ably/ably-go/pull/172#issuecomment-718666302) (which I'm working on now). But they showcase how users are likely to encounter a connection error: by trying to use the connection for publishing, subscribing or some other operation.

I can revert that commit for now so that this PR is mergeable, or let the PR blocked until those issues are fixed and then rebase.

Reply:
> See cbdaa10.

Great thanks 👍 

> I can revert that commit for now so that this PR is mergeable, or let the PR blocked until those issues are fixed and then rebase.

I think we can merge this and just focus on merging the other PR and related fixes before we share this updated README with people.